jdbc/src/main/java/org/apache/metamodel/jdbc/SqlKeywords.java

```diff
@@ -58,6 +58,8 @@
         KEYWORDS.add("DELETE");
         KEYWORDS.add("AND");
         KEYWORDS.add("OR");
+        KEYWORDS.add("END");
```

Comment:
I don't worry at all about the performance here because it is still a pretty small set and it happens just once. I worry a bit about maintaining this keyword list though, but that's a conversation that maybe goes beyond the scope of this little fix. I will merge it in.